addons/api/addon/models/worker.js

```diff
@@ -0,0 +1,27 @@
+import GeneratedWorkerModel from '../generated/models/worker';
+import { fragment } from 'ember-data-model-fragments/attributes';
+
+export default class WorkerModel extends GeneratedWorkerModel {
+  // =attributes
+
+  /**
+   * @type {[FragmentTagModel]}
+   */
+  @fragment('fragment-tag', {
```

Comment:
Should these `fragment`s be in the generated model? I noticed all of our models that use fragments have them in the normal models, is there a reason for this?

Reply:
@ZedLi 
I brought this topic to slack, the usage of fragments is an old friend of us, so wanted to bring up this to discussion 😃